manifests/server/access_wrapper.pp

```diff
@@ -10,35 +10,64 @@
 #
 # [*acl*]
```

Comment:
we should also update this to puppet-strings soonish

Reply:
I saw this yesterday and bravely decided to go look away :laughing: